mkosi/run.py

```diff
@@ -230,6 +230,9 @@ def run(
             TERM=os.getenv("TERM", "vt220"),
         ) | env
 
+    if kwargs.get("shell"):
+        cmd = [shlex.quote(s) for s in cmd]
```

Comment:
Do we actually use `shell` anywhere? If not I wouldn't bother adding this until we do

Reply:
We don't, but I stumbled over the `shlex.join` for printing the cmdline in one of the run functions, which is a bit performative, but then I thought that this is a potential footgun, since we pass `kwargs` through.

Reply:
This wouldn't even work right, since we have to join the command to a string when shell is used. I'd just drop this.